prisma/schema.prisma

```diff
@@ -10,19 +10,83 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum StationType {
+  Fixed
+  Mobile
+}
+
+type Audit {
+  created_at  DateTime?  @default(now())
+  created_by  DateTime?  @default(now())
+  updated_at  DateTime?  @default(now())
+  updated_by  DateTime?  @default(now())
+  deleted_at  DateTime?  @default(now())
+  deleted_by  DateTime?  @default(now())
+}
```

Comment:
It's good but some points:
- This will have to be a Model (the library is deprecated);
- Just created_at has a @default(now())
- all the: created_by, updated_by and deleted_by have a relation one to one with User model
- the default value of the others are null

Reply:
And since this is going to be a Model, each other Model is going to have a relation one to one with this model.